### ISSUE TITLE
[Bug]: Cannot access describe results

### ISSUE THREAD

**Issue by cosmincatalin** [mlrun/mlrun#2373]
### MLRun Version checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the latest version of the MLRun Kit.

### Reproducible Example

```python
import mlrun

proj = mlrun.new_project("sk-project-001", "./", init_git=True, overwrite=True)
describe = proj.set_function('hub://describe')
describe \
    .run(params={'label_column': 'label'},
         inputs={"table": "https://raw.githubusercontent.com/vizydrop/data-samples/master/flowers.csv"})
```

### Issue Description

```
pip-tools
mlrun==1.1.0
sklearn
```

![image](https://user-images.githubusercontent.com/525590/189516204-36a7990f-d76d-4604-a02d-345188629fee.png)

Running the code against the locally installed K8 Helm finishes successfully. However, there are two issues.

1. When navigating to http://localhost:30060/mlrun/projects/sk-project-001/datasets, the page immediately becomes blank. In the Chrome debugger I can see this

![image](https://user-images.githubusercontent.com/525590/189516119-26fc6273-e84a-4d19-99f7-82f304a7980f.png)

2. I can see that artifacts are produced, but they are not accessible

![image](https://user-images.githubusercontent.com/525590/189516262-4e662b6e-f9ef-4d8b-991d-3ba55b6aa1db.png)

### Expected Behavior

1. I can see whatever I am supposed to see in the datasets tab
2. I can access the artifacts produced by what appears to be a successful job

### Python Version

3.9

### MLRun Version

1.1.0

**Comment by Tankilevitch** [comment]
Hey @cosmincatalin nice to meet you!
So this is as a result of using newer mlrun client version with older mlrun api version. 

please align your mlrun client version: 
from your client:
```
pip install -U mlrun==1.0.6
```
Then remove the project using the UI.

**Comment by cosmincatalin** [comment]
Hi @Tankilevitch , thank you, works like charm now.
Can you please tell me how I'm able to figure out the API version, and is there a way to use `1.1.0`?

**Comment by Tankilevitch** [comment]
Glad to hear @cosmincatalin :) 
So there are few options to find the API version: 
1. From client:
```
import mlrun
print(mlrun.get_run_db().server_version)
```
2. From kubernetes ( the image contains the version suffix )
```
kubectl -n mlrun get deploy mlrun-api -oyaml | grep image:
```

There was a new bug introduced in 1.1.0 which will be solved 1.1.1 ( it affects the routing of the UI ).
Instead of accessing the `<UI-URL>` you will need to access `<UI-URL>/mlrun`, this will be solved in 1.1.1 so I preferred not disrupt the way users access the UI and wait until 1.1.1. 

If this doesn't bother you, and you acknowledge that this behavior is transient only for that version. 
Use the following commands to upgrade to 1.1.0:
```
helm repo update
helm upgrade --install --reuse-values mlrun-kit v3io-stable/mlrun-kit --set mlrun.api.image.tag=1.1.0 --set mlrun.ui.image.tag=1.1.0
```
and then from the client we will need trigger migrations which actually apply the database version updates
```
import mlrun
mlrun.get_run_db().trigger_migrations()
```
